server/controllers/cohort.js

```diff
@@ -422,7 +422,7 @@ const addMember = (req, resp) => {
       return User.findOne({ email: sanitize(email) }).exec();
     })
     .then(user => {
-      if (!user) {
+      if (!user || user.isActive === false) {
```

Comment:
`!user || !user.isActive` - do not compare boolean value to `true` or `false`

Reply:
I think that in this case, I have to compare to false because of users authenticated via Google - they don't have property isActive. For them, it will be `undefined` so this comparison will be true then. Or I can check: ` if(!user || (!idFromProvider && !user.isActive)`

Reply:
undefined is also a falsy value so you will get true if isActive filed is empty, undefined or null
!null = true
!undefined = true
!'' = true

Reply:
if(!user || (!idFromProvider && !user.isActive)

this is good I think